thentos-core/src/Thentos/Sybil/Captcha.hs

```diff
@@ -0,0 +1,15 @@
+{-# LANGUAGE OverloadedStrings    #-}
+
+module Thentos.Sybil.Captcha where
```

Comment:
I think `Thentos.Sybil.Captcha` should just be `Thentos.Captcha`, unless there is a point to the extra indirection that I'm missing.